src/Css.elm

```diff
@@ -3092,6 +3104,21 @@ alignSelf fn =
     getOverloadedProperty "alignSelf" "align-self" (fn lengthForOverloadedProperty)
 
 
+{-| Sets [`justify-content`](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content).
+
+  **Note:** `auto` is not currently supported here. If you need to set this property to `auto`,
+  use this workaround:
+
+  property "justify-content" "auto"
+
+  If this is annoying, please file an issue, so adding support for "auto"
+  can be prioritized!
+-}
+justifyContent : (ExplicitLength IncompatibleUnits -> Mixin) -> Mixin
+justifyContent fn =
+    getOverloadedProperty "justifyContent" "justify-content" (fn lengthForOverloadedProperty)
```

Comment:
Ah so as it turns out `auto` is not an option for `justify-content`. It is only a value for `align-self`. I've removed the comment regarding the `auto` property.